projects/packages/newsletter/package.json

```diff
@@ -32,7 +32,7 @@
 		"extends @wordpress/browserslist-config"
 	],
 	"dependencies": {
-		"@wordpress/dataviews": "10.2.0",
+		"@wordpress/dataviews": "11.0.0",
```

Comment:
@dsas, I'm assuming this dependency is here by mistake? I don't think the Newsletters page actually uses DataViews.